test-data/unit/cmdline.test

```diff
@@ -400,9 +400,9 @@ bla bla
 [file error.py]
 bla bla
 [out]
+normal.py:2: error: Unsupported operand types for + ("int" and "str")
```

Comment:
Ah, so this is an example where the errors for different files are interspersed. Nifty! :-)